### DISCUSSION TITLE
What mongodb driver does Prisma use?

### DISCUSSION THREAD

**Discussion by Milo123459** [prisma/prisma#12886] [Q&A] [answered]
And what bson parser does it use?

**Comment by janpio** [comment, marked as answer]
https://github.com/mongodb/mongo-rust-driver

(Exact current version: https://github.com/prisma/prisma-engines/blob/73e60b76d394f8d37d8ebd1f8918c79029f0db86/query-engine/connectors/mongodb-query-connector/Cargo.toml#L13)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
